core/internal/runfiles/uploader.go

```diff
@@ -51,7 +56,20 @@ func (u *uploader) Process(record *service.FilesRecord) {
 		return
 	}
 
-	// TODO
+	nowFiles := make([]string, 0)
+
+	for _, file := range record.GetFiles() {
+		switch file.GetPolicy() {
+		case service.FilesItem_NOW:
+			nowFiles = append(nowFiles, file.GetPath())
+		case service.FilesItem_LIVE:
+			// TODO
+		case service.FilesItem_END:
+			// No-op. All files are uploaded at the end by default.
```

Comment:
i don't remember what we did before, but the plan is to also upload the now files in the end? no right? only live and end?

Reply:
good point! I still need to add some policy tracking to avoid re-uploading "now" files at the end